### ISSUE TITLE
Support for puppet v4

### ISSUE THREAD

**Issue by OrangeCrush** [n1tr0g/golja-gnupg#25]
Has anyone looked at what it would take to upgrade this module to be compatible with puppet v4?

The documentation on the puppet forge suggests that the module is not supported / tested past v4.0

**Comment by n1tr0g** [comment]
that's correct :( Its in my TODO list. I will try to have a go today. My concern is the custom type/provider. Maybe will have to be adjusted for V4, which means the next release will be a breaking one.
